server/s3_addb_map_auto.c

```diff
@@ -147,6 +147,22 @@ const char* g_s3_to_addb_idx_func_name_map[] = {
     "S3HeadBucketAction::send_response_to_s3_client",
     "S3HeadObjectAction::send_response_to_s3_client",
     "S3HeadServiceAction::send_response_to_s3_client",
+    "S3MultiObjectCopyAction::check_part_details",
```

Comment:
S3<Operation-type> followed elsewhere, though somewhere 'Multi' & other places 'Multipart' used, so lets put that will keep things clearer i.e **S3PutMultipartObjectCopyAction**

Reply:
Updated class name to `S3PutMultipartCopyAction`